src/components/SavedTrip.js

```diff
@@ -21,6 +22,13 @@ function SavedTrip() {
       <TripDetails name="activities" text={singleTrip.activities} />
       <TripDetails name="restaurants" text={singleTrip.restaurants} />
       <TripDetails name="notes" text={singleTrip.notes} />
+      <Image
+        cloudName="dyjecx1wm"
```

Comment:
Just out of curiosity, where do you get the `cloudName` from?
Maybe this is something that could be an environmental variable?

Reply:
The cloudname is my personal cloud name I got from cloudinary.
It needs to be in there, so the image can get fetched again from this specific folder.